File: Data Structures & Algorithms/longest-common-prefix/submission-0.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        sort(strs.begin(),strs.end());
        string k="";
        int n=strs.size();
        for(int i=0;i<strs[0].size();i++){
            if(strs[0][i]==strs[n-1][i]){
                k+=strs[0][i];
            }
            else{
                break;
            }
        }
        return k;
    }
};

File: Data Structures & Algorithms/string-encode-and-decode/submission-0.py
class Solution:

    def encode(self, strs: List[str]) -> str:
        res=""
        for i in strs:
            res+= i+"#"
        return res

    def decode(self, s: str) -> List[str]:
        a=s.split('#')
        a.remove(a[-1])
        return a
